tensorboard/plugins/npmi/BUILD

```diff
@@ -3,6 +3,7 @@
 # models. nPMI is a technique presented in a paper (https://svn.spraakdata.gu.se/repos/gerlof/pub/www/Docs/npmi-pfd.pdfhttps://svn.spraakdata.gu.se/repos/gerlof/pub/www/Docs/npmi-pfd.pdf)
 # presented by Gerlof Bouma.
 
+load("@rules_python//python:defs.bzl", "py_binary", "py_library", "py_test")
```

Comment:
I think this line may break the internal sync.  All other build files assume these are loaded ambiently, and there isn't currently any transformation that updates these lines at sync time.

Please remove?